Doc/howto/descriptor.rst

```diff
@@ -1413,7 +1421,7 @@ It is not possible to create an exact drop-in pure Python version of
 ``__slots__`` because it requires direct access to C structures and control
 over object memory allocation.  However, we can build a mostly faithful
 simulation where the actual C structure for slots is emulated by a private
-``_slotvalues`` list.  Reads and writes to that private structure are managed
+``_slot_values`` list.  Reads and writes to that private structure are managed
```

Comment:
Let's let this as is.  It is just a question of taste.  We normally only add a extra underscore if the combined word is hard to mentally separate.

Reply:
Alright.